timelock-agent/src/main/java/com/palantir/timelock/paxos/AsyncTimeLockServicesCreator.java

```diff
@@ -82,16 +83,16 @@ public TimeLockServices createTimeLockServices(
     private static AsyncTimelockService createRawAsyncTimelockService(
             String client,
             Supplier<ManagedTimestampService> timestampServiceSupplier) {
-        ScheduledExecutorService reaperExecutor = Executors.newSingleThreadScheduledExecutor(
-                new ThreadFactoryBuilder()
+        ScheduledExecutorService reaperExecutor = (ScheduledExecutorService) new InstrumentedExecutorService(
```

Comment:
are we sure this cast is safe?

Reply:
this cast is not safe, there's a separate `InstrumentedScheduledExecutorService` for wrapping `ScheduledExecutorService`s, see http://metrics.dropwizard.io/3.1.0/apidocs/com/codahale/metrics/InstrumentedScheduledExecutorService.html